style/styles.css

```diff
@@ -42,7 +44,61 @@ header {
   color: #6070ff;
 }
 
-.menu-items {
+/*
+.menu{
+  display: none;
+}
+*/
+
+.menu {
+  position: fixed;
+  transform: translateY(-100%);
+  transition: transform 0.3s;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 99;
+  background: #6070ff;
+  list-style: none;
+  opacity: 0.9;
+  padding-top: 2rem;
+}
```

Comment:
- [ ]  I think here you need to blur the background to match the Figma design as much as possible ✅✅